tools/hardware/killerbee_msfrelay.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
```

Comment:
Looks like this module doesn't work with python3, as it relies on `
from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer` which doesn't work on python3

Python2:
```
$ python2
Python 2.7.16 (default, Jun  5 2020, 22:59:21) 
[GCC 4.2.1 Compatible Apple LLVM 11.0.3 (clang-1103.0.29.20) (-macos10.15-objc- on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
>>> HTTPServer
<class BaseHTTPServer.HTTPServer at 0x10cdf3600>
>>> 
```

Python3:
```
$ python3
Python 3.9.2 (default, Mar 26 2021, 23:27:12) 
[Clang 12.0.0 (clang-1200.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'BaseHTTPServer'
>>> 
```

Reply:
I'll get that reverted

Reply:
@cgranleese-r7 I think if every module is meant to be python3, and our error message explicitly states you need python3, then this module most likely needs converted over to python3 - or special cased. Reverting this file alone isn't enough IMO 👀

Reply:
@adfoster-r7 Ok, I'll have to look into how to go about doing that, as it's not something I've done before 👍

Reply:
Looked into this and it seems